docs/src/ToricVarieties/AlgebraicCycles.md

```diff
@@ -7,9 +7,68 @@ Pages = ["AlgebraicCycles.md"]
 ```
 
 
-# Algebraic Cycles
+# The Chow ring and algebraic Cycles
 
-## Special attributes of toric varieties
+Algebraic cycles are elements of the Chow ring.
```

Comment:
Perhaps you could define what a Chow ring is, or at least sketch it, or give a ref to a source that uses convention that match what you are doing here?

I also noticed that we already have some code dealing with Chow rings, from the Matroids code?. I am totally out of my depth here, but I wonder if these various definition fit together? Perhaps the responsible people could coordinate? CC @bschroter @LukasKuehne 

Some places referencing them include:
- `docs/src/Combinatorics/matroids.md`
- `src/Combinatorics/Matroids/ChowRings.jl`
-

Reply:
Similar to the above. I have added more details. (Aside, this ring is up to isomorphism unique for a toric variety. This is similar in spirit to the `coordinate_ring` of a variety.)

On first glance, I found it surprising to see this for `Matroids` and `Combinatorics.` Then again, I could imagine (not 100% sure though) that one could "strip" the toric variety and define the (or a?) Chow ring just based on combinatorics. This would be similar in spirit to the Stanley-Reisner ideal... But I must say I have not checked in detail just yet.

Happy to hear more from the responsible people.

Reply:
@HereAround you are right. One doesn't need a toric variety to define the Chow Ring. The implementation for matroids goes back to the construction of Feichtner and Yuzvinsky and is a key element in all of June Huh's results (for which we received the fields medal).

Reply:
I see. Good to know.

Similar as @fingolfin I wonder then if it would make sense to "marry" the approaches.

To get this started (sorry - I am pretty sure your docu tells but I have not yet checked due to a conference): What information (of the toric variety) do you need to construct the Chow ring?

Right now I take the quotient of the Cox ring hy the sum of the Stanley- Reisner and the irrelevant ideal.

Reply:
Probably the best is to first keep it separate and then meet to discuss if/and how one could merge those into one.

Reply:
Ok. I open an issue (https://github.com/oscar-system/Oscar.jl/issues/1475) so that we do not forget this.